### ISSUE TITLE
Unhandled Promise Rejection after wake up from sleep (windows)

### ISSUE THREAD

**Issue by alexn-s** [johnlindquist/kit#969]
i received this error twice after opening my laptop (sleep -> wakeup).
os: windows

![22-12-07_12-40-05_678 _ Kit _ Kit](https://user-images.githubusercontent.com/60710873/206269844-6825d9a4-ea33-416c-b323-c8e687a7e013.png)

```
Unhandled Promise Rejection
Error: read ECONNRESET
    at Function.Ie.from (C:/Users/aschu/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:14691)
    at y.<anonymous> (C:/Users/aschu/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:32631)
    at y.emit (node:events:526:28)
    at ClientRequest.l.forEach.p.<computed> (C:/Users/aschu/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:988927)
    at ClientRequest.emit (node:events:526:28)
    at TLSSocket.socketErrorListener (node:_http_client:442:9)
    at TLSSocket.emit (node:events:526:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

do you need more information or some log files?

ps: do you think you can push the fix for #965 this week (looking forward to replace espanso with kit)?

**Comment by johnlindquist** [comment]
The log would be great. I'm building the new build now, but want to fix this if you can send me the ~/.kit/logs/kit.log

**Comment by alexn-s** [comment]
[kit.log](https://github.com/johnlindquist/kit/files/10179517/kit.log)

attached

**Comment by alexn-s** [comment]
should be fixed by v1.39.16 (#970)
